README.md

```diff
@@ -130,4 +130,8 @@ Join the [Discord server](discord.gg/melkeydevhouse)! We're always around to hel
 
 You can also probably catch Melkey working live on [Twitch](twitch.tv/melkey)! 
 
-Pop in and say hi! ♡
\ No newline at end of file
+Pop in and say hi! ♡
+
+## License
+
+Licensed under [./MIT License]
```

Comment:
Still not quite there:
```
Licensed under [MIT License](./LICENSE)
```

how about we try something like this?

Reply:
Thank you for the learning opportunity!